todo/views.py

```diff
@@ -0,0 +1,75 @@
+from django.shortcuts import get_object_or_404, redirect
+from django.urls import reverse_lazy
+from django.views import generic, View
+
+from todo.forms import TagForm, TaskForm
+from todo.models import Task, Tag
+
+
+class TaskListView(generic.ListView):
+    model = Task
+    context_object_name = "task_list"
+    template_name = "todo/task_list.html"
+    paginate_by = 3
+    queryset = Task.objects.prefetch_related("tags")
+
+
+class TaskCreateView(generic.CreateView):
+    model = Task
+    form_class = TaskForm
+    template_name = "todo/task_form.html"
+    success_url = "http://127.0.0.1:8000/task/create"
+
+
+class TaskUpdateView(generic.UpdateView):
+    model = Task
+    form_class = TaskForm
+    template_name = "todo/task_form.html"
+    success_url = reverse_lazy("todo:task-list")
+
+
+class TaskDeleteView(generic.DeleteView):
+    model = Task
+    fields = "__all__"
```

Comment:
You should remove `fields = "__all__"` because it is unnecessary for `DeleteView` classes in Django. The `DeleteView` doesn't use form fields like `CreateView` or `UpdateView`. Instead, it uses the `get_object` method to retrieve the object to be deleted and then confirms the deletion through a form submission.